server/index.js

```diff
@@ -116,6 +116,38 @@ app.post('/api/cart', (req, res, next) => {
     .catch(err => next(err));
 });
 
+app.post('/api/orders', (req, res, next) => {
+  if (!req.session.cartId) {
+    res.status(400).json({
+      error: 'there is not cardId'
+    });
+  }
+  const { name, creditCard, shippingAddress } = req.body;
+  if (name === undefined || creditCard === undefined || shippingAddress === undefined || typeof creditCard !== 'number') {
+    res.status(400).json({
```

Comment:
Be careful to `return` so that your server does not try to respond twice and crash.

Reply:
got it!